### ISSUE TITLE
Layers Diff

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#6]
# Layers
- [ ] Convolution Layer
- [ ] Pooling Layer
- [ ] ROIPooling Layer
- [ ] FullyConnected Layer
- [ ] ReLU Layer
- [ ] Activation Layer
- [ ] SoftMax layer
- [ ] Deconvolution Layer
- [ ] Local Response Normalization (LRN) layer
- [ ] Concat Layer
- [ ] Split Layer
- [ ] Reshape Layer
- [ ] Eltwise Layer
- [ ] ScaleShift Layer
- [ ] Crop Layer
- [ ] Batch Normalization Layer
- [ ] Normalize Layer
- [ ] Tile Layer
- [ ] Permute Layer
- [ ] PriorBox Layer
- [ ] SimplerNMS layer
- [ ] DetectionOutput Layer
- [ ] Memory / Delay Object layer
- [ ] Clamp Layer
- [ ] ArgMax Layer
- [ ] PSROIPooling Layer
- [ ] GRN Layer
- [ ] PReLU Layer
- [ ] PriorBoxClustered Layer
- [ ] MVN Layer
- [ ] CTCGreadyDecoder Layer
- [ ] Proposal Layer
- [ ] Resample Layer

# Template
- 作用
- 实现方式

**Comment by junxnone** [comment]
# Convolution Layer

**Comment by junxnone** [comment]
# Pooling Layer
